--- ruby-mixlib-authentication-1.4.0/mixlib-authentication.gemspec~	2016-06-02 19:29:52.000000000 +0300
+++ ruby-mixlib-authentication-1.4.0/mixlib-authentication.gemspec	2016-06-02 19:31:40.599501191 +0300
@@ -20,6 +20,6 @@
   s.files = %w(LICENSE README.rdoc Gemfile Rakefile NOTICE) + Dir.glob("*.gemspec") +
       Dir.glob("{lib,spec}/**/*", File::FNM_DOTMATCH).reject {|f| File.directory?(f) }
 
-  %w(rspec-core rspec-expectations rspec-mocks).each { |gem| s.add_dependency gem, "~> 3.2" }
+  %w(rspec-core rspec-expectations rspec-mocks).each { |gem| s.add_development_dependency gem, "~> 3.2" }
   s.add_development_dependency "rake", "~> 10.4"
 end
